server/src/db/migrations/20230207171703_addUpvotesAndDownVotes.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("reviews", table => {
+    table.integer("votes").notNullable().defaultTo(0)
+  })
+}
+
+/**
+ * @param {Knex} knex
+ */
+exports.down = (knex) => {
+  return knex.schema.table("reviews", table => {
```

Comment:
this migration file should be deleted - we want Votes to be its own table, not a property on a Review object.